### ISSUE TITLE
Accessibility statement

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#644]
A few people have asked for an accessibility statement for Numbas. Ideally, we should have separate statements for the runtime and the editor.

The W3C has [a page on preparing an accessibility statement](https://www.w3.org/WAI/planning/statements/). It says a statement should include:

* A commitment to accessibility for people with disabilities
* The accessibility standard applied, such as WCAG 2.1
* Contact information in case users encounter problems
* Any known limitations, to avoid frustration of your users
* Measures taken by your organization to ensure accessibility
* Technical prerequisites, such as supported web browsers
* Environments in which the content has been tested to work
* References to applicable national or local laws and policies

**Comment by christianp** [comment]
[Accessibility statement for MyLab Math](https://www.pearsonmylabandmastering.com/northamerica/mymathlab/accessibility/index.html)

**Comment by christianp** [comment]
@chrismgraham would you mind reading the draft statement in [accessibility.md](https://github.com/numbas/Numbas/blob/master/accessibility.md)?

**Comment by christianp** [comment]
Now in the docs
